README.md

```diff
@@ -4,6 +4,30 @@
 
 Python package for searching for housing on Craigslist.
 
+-   Author: Ela Bandari, Junting He, Ling (Elina) Lin, Alex Truong
+
+
+## Overview
+
+Hunting for rental can be an exhausting and frustrating experience in Canada, but this process can be made easy with a simple installation of our package. This Python package intends to facilitate house hunting by scraping the listing information from Craigslist and organizing the extracted data for the users. Instead of having to manually go on the website to catch up with individual new listings, the users will be updated through email with new listings as per their selection criteria. 
+
+## Functions
+
+| Function Name | Input | Output | Description |
+|-----------|------------|---------------|------------------|
+| web_scraper | url | csv file | Scrape data from rental websites into a csv file.|
+| data_cleaner | csv file location | csv file | Clean the extracted data |
+| data_filter | csv file, min_price, max_price, sqrt_ft, no_bathroom, no_bedroom, neighbourhood | csv file | Filter the cleaned data set based on user inputs|
```

Comment:
Minor suggestion. I wonder if num_bathroom and num_bedroom are better variable names?

Reply:
Yes I agree with that!